### PR TITLE
Improve winrm docs

### PR DESCRIPTION
**Pull request by aledsage**
@bostko @ygy can you review this please?

### CODE REVIEW

**Comment by aledsage** [comment]
@nakomis can you review as well please?

File: docs/guide/yaml/winrm/index.md [hunk 1]
```diff
@@ -82,7 +82,8 @@ For example - here is a blueprint:
         region: us-west-2
         identity: AKA_YOUR_ACCESS_KEY_ID
         credential: <access-key-hex-digits>
-        imageNameRegex: Windows_Server-2012-R2_RTM-English-64Bit-Base
+        imageNameRegex: Windows_Server-2012-R2_RTM-English-64Bit-Base-.*
+        imageOwner=801119661308
```
Comment (bostko) [inline docs/guide/yaml/winrm/index.md]: imageOwner: 801119661308

File: docs/guide/yaml/winrm/index.md [hunk 2]
```diff
@@ -115,40 +116,172 @@ This is only a very simple example. A core complex example can be found in the [
...
+Brooklyn will run powershell scripts by making a WinRM call over HTTP. For most scripts this will work, however for
+some scripts (e.g. MSSQL installation), this will fail even if the script can be run locally (e.g. by using RDP to
+connect to the machine and running the script manually)
+
+For example in the case of MS SQL server installation, there was no clear indication of why this would not work. 
+The only clue was a security exception in the installation log.
+
+When a script is run over WinRM over HTTP, the credentials under which the script are run are marked as
+'remote' credentials, which are prohibited from running certain security-related operations. The solution was to obtain
+a new set of credentials within the script and use those credentials to exeute the installer, so this:
```
Comment (bostko) [inline docs/guide/yaml/winrm/index.md]: type "execute"

File: docs/guide/yaml/winrm/index.md [hunk 3]
```diff
@@ -115,40 +116,172 @@ This is only a very simple example. A core complex example can be found in the [
...
+
+ The redirect can be achieved by using the `Start-Process` scriptlet:
+
+    Start-Process D:\setup.exe -ArgumentList "/ConfigurationFile=C:\ConfigurationFile.ini" -RedirectStandardOutput "C:\sqlout.txt" -RedirectStandardError "C:\sqlerr.txt" -PassThru -Wait
+
+The `-ArgumentList` is simply the arguments that are to be passed to the executable, `-RedirectStandardOutput` and
+`RedirectStandardError` take file locations for the output (if the file already exists, it will be overwritten). The
+`-PassThru` argument indicates that PowerShell should write to the file *in addition* to the console, rather than
+*instead* of the console. The `-Wait` argument will cause the scriptlet to block until the process is complete.
+
```
Comment (bostko) [inline docs/guide/yaml/winrm/index.md]: Additions I found:
You can also check the logs inside `%programfiles%\Microsoft SQL Server\130\Setup Bootstrap\Log\`  for more information check https://msdn.microsoft.com/en-us/library/ms143702.aspx.

Other system errors in Windows are usually reported in the Windows Event Log check https://technet.microsoft.com/en-us/library/cc766042.aspx for more information

**Comment by bostko** [comment]
Good fixes. I like everything to be in one place.
OK to merge!

**Comment by aledsage** [comment]
Thanks @bostko - merging.

**Pull request merged**